wherehows-frontend/app/controllers/Application.java

```diff
@@ -54,6 +54,7 @@
   private static final Boolean IS_INTERNAL = Play.application().configuration().getBoolean("linkedin.internal", false);
   private static final String JIT_ACL_WHITELIST =
       Play.application().configuration().getString("linkedin.jit.acl.whitelist", "");
+  private static final String WHZ_DS_OWNERSHIP_TAB = System.getenv("WHZ_DS_OWNERSHIP_TAB");
```

Comment:
Normally we get configs for Play application.conf file. You can set the conf file to take env var instead, (or fix a value in testing). Easier to test and more friendly to Open source